posts/2022/xarray-groupby-vs-geocat-climatology.ipynb

```diff
@@ -0,0 +1,423 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #3.        ds.resample(time='QS-DEC').mean().groupby('time.month').mean()
</pre><p>Hmm this seems quite incorrect, calculating the mean twice is usually a bad idea. To do it this way you would have to calculate the <code>sum</code> and <code>count</code> individually then divide to get <code>mean</code>.  </p><p> </p><p>I think you just want <code>ds.groupby("time.season").mean().sel(season=["DJF", "MAM", "JJA", "SON"])</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/esds/pull/216/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>